### ISSUE TITLE
About Continuous Diffusion Implementation

### ISSUE THREAD

**Issue by CIntellifusion** [kwsong0113/diffusion-forcing-transformer#13]
Hi authors, thanks for your opensource of this excellent work. 

I am training contious dfot and discrete dfot on my own set. I found the discrete Doft works properly with a training loss about 0.25. But the continous Dfot only has a training loss about 0.025( 10x lower) while the inference seems to be wrong. 

I have checked the sampling function in dfot.py and it uses ' def model_predictions(self, x, k)' to generate video.  

The question is : ContinuousDiffusion is inherent from DiscreteDiffusion, but without rewrite predict_start_from_v and predict_noise_from_v which may cause error during sampling? 

the forward function of  ContinuousDiffusion  has this:
```
        # v-prediction
        v_pred = self.model(x_t, self.precond_scale * logsnr, external_cond)
        noise_pred = alpha_t * v_pred + sigma_t * x_t
        x_pred = alpha_t * x_t - sigma_t * v_pred

        loss = F.mse_loss(noise_pred, noise.detach(), reduction="none")
```

I think it should be used during training by rewrite predict_noise_from_v?

**Comment by kwsong0113** [comment]
We implemented ⁠`ContinuousDiffusion` with minimal functionality:
- At training time, it uses its own `forward` method
- At sampling time, it simply modifies `model_prediction` from `DiscreteDiffusion`, by converting a timestep from the sampling schedule to logsnr before the model's forward pass

Thus, to make continuous diffusion work, the training schedule and sampling schedule should be be identical -- please take a look at `configurations/dataset_experiment/realestate10k_video_generation.yaml` and `configurations/shortcut/diffusion/continuous.yaml` (Essentially, you can replicate [how we train our Re10K model](https://github.com/kwsong0113/diffusion-forcing-transformer/wiki/Training#realestate10k))

Also, it's normal for diffusion model's loss to vary in scale and range depending on parameterization & weighting (we also had a small loss range when training our Re10K model).
